workshops.qmd

```diff
@@ -116,20 +118,24 @@ Still a work-in-progress, but should ask questions about:
     -   Presented to: \[audience\]
     -   Description: \[description of workshop as in email announcements\]
 -   If the video is unedited from the Zoom recording, you can upload the exported .vtt file for closed captions with timings
--   If possible, include thumbnail photo from a relevant slide
+-   If possible, include thumbnail from a relevant slide
 -   Add to the 'Workshops' playlist
 -   Once it's uploaded (may take a while), including the HD version, check that the video is clear enough to see text on screen
--   Advertise video upload via Twitter!
+-   Advertise video upload via Mastodon!
+
+::: callout-tip
+One benefit of editing the video on YouTube studio is that you can upload the .vtt file with a timed transcript along with the original video and then when you trim the video, the subtitles will be trimmed along with it.
+The .vtt file from Zoom has participant names in it and is probably more accessible than the auto-generated YouTube subtitles for that reason.
+:::
 
 ### Share materials
 
 -   Submit a pull request to [cct-datascience/CALS-workshops](https://github.com/cct-datascience/CALS-workshops) with notes and/or slides
 -   Update workshop info on our [workshops website page](https://datascience.cct.arizona.edu/workshops)
-    -   Under **Category**, add "ALVSCE Workshop Wednesdays" option
+    -   Under **Category**, add "ALVSCE Workshop Wednesdays" option (or other relevant category)
     -   Under **Additional Links**, add slides, notes, etc., with corresponding URLs
-    -   Under **Recording**, add URL of YouTube recording of workshop and **Title** to "View Recording"
-    -   Check box under **Archive**
```

Comment:
Confirmed that archiving is now automated and this checkbox no longer exists

Reply:
This maybe didn't work for ggplot part 3, so I'm asking Hannah.